### PR TITLE
release: v0.6.0

### PR DESCRIPTION
**Pull request by ewanharris**
## Description

Releasing the OpenTelemetry metrics changes. The first commit is mostly some whitespace changes, license headers, and reordering of things as a consequence of how this change had to be integrated into the generator.

## References

Generated from https://github.com/openfga/sdk-generator/pull/383

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

**Review by rhamzeh — APPROVED** [review]
@ewanharris do you think we can add the metrics you have documented in the PR to the README? (can be in a follow up PR, does not block release)

Even better if we can have a metrics section in the README that links to docs/OTEL_METRICS.md so as not to clutter the readme.

Please also add a note in both that the metrics are not sent if not configured, and will go to the user's OTEL collector, not to the OpenFGA team, in case the wording scares folks.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.6.0
+
+### [0.5.0](https://github.com/openfga/js-sdk/compare/v0.5.0...v0.6.0) (2024-06-28)
+- feat: add opentelemetry metrics reporting
```
Comment (rhamzeh) [inline CHANGELOG.md:6]:
```suggestion
- feat: add opentelemetry metrics reporting (#117)
```

**Comment by ewanharris** [comment]
@rhamzeh, ah yeah I shoulda thought of that. I'll add it in this PR a it'll then be on the npm docs too

**Comment by ewanharris** [comment]
@rhamzeh Added some docs in 847c456fbe20e5800ae53eee1867bb1f579b75ed, pushing the SDK generator changes momentarily.

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**